src/ProjectTemplates/Web.ProjectTemplates/content/EmptyWeb-CSharp/Program.cs

```diff
@@ -1,26 +1,14 @@
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
+using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
 
-namespace Company.WebApplication1
-{
-    public class Program
-    {
-        public static void Main(string[] args)
-        {
-            CreateHostBuilder(args).Build().Run();
-        }
+await using var app = WebApplication.Create(args);
 
-        public static IHostBuilder CreateHostBuilder(string[] args) =>
-            Host.CreateDefaultBuilder(args)
-                .ConfigureWebHostDefaults(webBuilder =>
-                {
-                    webBuilder.UseStartup<Startup>();
-                });
-    }
+if (app.Environment.IsDevelopment())
+{
+    app.UseDeveloperExceptionPage();
 }
+
+app.MapGet("/", (Func<string>)(() => "Hello World!"));
+
+await app.RunAsync();
```

Comment:
Program+Startup before: 66 lines
after: 14 lines. 

It's a lot emptier than it used to be 😁.